client/my-sites/stats/geochart/index.jsx

```diff
@@ -18,17 +17,51 @@ module.exports = React.createClass( {
 	visualization: null,
 
 	propTypes: {
-		data: React.PropTypes.array.isRequired,
-		dataList: React.PropTypes.object.isRequired
+		data: PropTypes.array.isRequired,
+		dataList: PropTypes.object.isRequired
```

Comment:
You're already including `React`, why also `PropTypes` which is a property of react? (curious)